OpenShift/helm/nexus-iq/templates/deployment.yaml

```diff
@@ -3,51 +3,55 @@ kind: Deployment
 metadata:
   name: {{ template "iqserver.fullname" . }}
   labels:
-    app: {{ template "iqserver.fullname" . }}
+    {{- include "iqserver.labels" . | nindent 4 }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicaCount }}
```

Comment:
For our initial cases, I understood that we'd not want more than one instance, since the server's not likely friendly to that yet. Am I wrong? It's cool to have the option, I guess, but I'm interested to see what it does too.

Reply:
I'm following stock patterns at this point but I think you are spot on. We can open an issue and fix as a discreet change